frontend/src/components/App.js

```
@@ -0,0 +1,56 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import ComponentsComponent from './Components'
+import {componentsOperations} from './duck'
+import get from "lodash/get";
+
+const mapStateToProps = (state) => {
+    const {buildInfo, loggedUser} = get(state, "components")
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(indent)